source/SIL.AppBuilder.Portal.Frontend/src/data/containers/with-current-user.tsx

```diff
@@ -79,10 +78,14 @@ export function withCurrentUser() {
         this.fetchCurrentUser();
       }
 
+      isUserLocked = (response) => {
```

Comment:
403 won't always mean the user is locked. so I don't think it makes sense to always extract this to a method

Reply:
So if 403 is not the correct way to check if a user is Blocked should I try to read the error message and see if has "some keyword about blocked" ?

Reply:
checking for 403 is correct, but 403 will not always mean the user is locked

403 means forbidden